debian/ocf-atool.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
```

Comment:
You can rename this file to simply `debian/service` since it is the only/first binary package.